src/components/Prompt/Prompt.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {useEffect, useState} from 'react';
+
+import {Button, ButtonProps} from '@gravity-ui/uikit';
+import block from 'bem-cn-lite';
+
+import './Prompt.scss';
+
+const b = block('Prompt');
+
+export interface PromptProps {
+    // Prompt message
```

Comment:
I think this comment and comments below are unnecessary

Reply:
No, it's nice.In the future I plan to write comments like this in all places in package

Reply:
on this, I'd like to keep comments for clarity. There is not enough comments in the entire project. Without comments, you have to dig into the code to understand what it does, which slows you down a lot.

Reply:
In this case, why it's not in `/** comment */` format?